shared/charts/gpii-flowmanager/templates/gateway.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: {{ template "flowmanager.name" . }}-gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 443
+      name: https-{{ template "flowmanager.name" . }}
+      protocol: HTTPS
+    tls:
+      mode: SIMPLE
+      serverCertificate: /etc/istio/ingressgateway-certs/flowmanager-tls.crt
+      privateKey: /etc/istio/ingressgateway-certs/flowmanager-tls.key
```

Comment:
Where do these files come from? Are they managed by the injected istio sidecar container?

Reply:
These certs come from **cert-manager** originally, and they're merged into `istio-ingressgateway-certs` secret in `istio-system` namespace by **certmerge-operator** (that was deployed previously). This secret is then mounted in the `istio-ingressgateway` pods. (See certmerge-operator chart and https://github.com/gpii-ops/gpii-infra/blob/master/gcp/modules/certmerge-operator-crd/values.yaml for more details)